apps/ledger-live-desktop/release-notes.json

```diff
@@ -1,6 +1,14 @@
 [
+  {
```

Comment:
We should remove this file from the PR ?

Reply:
note very important to be honest as it is generated on build lld